src/ralph/backends/data/es.py

```diff
@@ -219,7 +219,9 @@ def read(  # noqa: PLR0912, PLR0913
             chunk_size (int or None): The chunk size when reading documents by batches.
                 If chunk_size is `None` it defaults to `DEFAULT_CHUNK_SIZE`.
             raw_output (bool): Controls whether to yield dictionaries or bytes.
-            ignore_errors (bool): Ignored.
+            ignore_errors (bool): If `True`, errors during the encoding operation
+                will be ignored and logged. If `False` (default), a `BackendException`
+                will be raised if an error occurs.
```

Comment:
unless I am mistaken, this behavior has not been changed and `ignore_errors`. Or did I miss smt ? 

at least, the warning message is still there ```"The `ignore_errors` argument is ignored"```

Reply:
Nice catch) Thanks) Have removed the `warning` message) You are right; the behavior has not changed, however, we do handle the `ignore_errors` argument, (even if it has no impact on the result) - updated the docstring to:
```
No impact as encoding errors are not expected in Elasticsearch results.
```